__tests__/connection.test.js

```diff
@@ -96,6 +99,78 @@ describe("/api/reviews/:review_id", () => {
 				expect(res.body.msg).toBe("Review_id not found");
 			});
 	});
+	test("PATCH - status 200 - Responds with an updated review object when passed a valid review_id and a valid inc_votes object", () => {
+		const incVotesObj = { inc_votes: 10 };
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(incVotesObj)
+			.expect(200)
+			.then((res) => {
+				expect(typeof res.body).toBe("object");
+				const review = res.body.review;
+				expect(review.length).toBe(1);
+				expect(typeof review[0].review_id).toBe("number");
+				expect(typeof review[0].title).toBe("string");
+				expect(typeof review[0].category).toBe("string");
+				expect(typeof review[0].designer).toBe("string");
+				expect(typeof review[0].owner).toBe("string");
+				expect(typeof review[0].review_img_url).toBe("string");
+				expect(typeof review[0].created_at).toBe("string");
+				expect(typeof review[0].votes).toBe("number");
+				expect(review[0].votes).toBe(11);
+			});
+	});
+	test("PATCH - status 200 - Responds with an updated review object when passed a valid review_id and a valid inc_votes object", () => {
+		const incVotesObj = { inc_votes: -11 };
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(incVotesObj)
+			.expect(200)
+			.then((res) => {
+				expect(typeof res.body).toBe("object");
+				const review = res.body.review;
+				expect(review.length).toBe(1);
+				expect(typeof review[0].review_id).toBe("number");
+				expect(typeof review[0].title).toBe("string");
+				expect(typeof review[0].category).toBe("string");
+				expect(typeof review[0].designer).toBe("string");
+				expect(typeof review[0].owner).toBe("string");
+				expect(typeof review[0].review_img_url).toBe("string");
+				expect(typeof review[0].created_at).toBe("string");
+				expect(typeof review[0].votes).toBe("number");
+				expect(review[0].votes).toBe(-10);
+			});
+	});
+	test("PATCH - status 400 - Responds with an invalid review_id error message when passed an invalid review_id", () => {
+		const incVotesObj = { inc_votes: 10 };
+		return request(app)
+			.patch("/api/reviews/one")
+			.send(incVotesObj)
+			.expect(400)
+			.then((res) => {
+				expect(res.body.msg).toBe("Invalid review_id");
+			});
+	});
+	test("PATCH - status 404 - Responds with an error message when passed a review_id which returns no results", () => {
+		const incVotesObj = { inc_votes: 10 };
+		return request(app)
+			.patch("/api/reviews/1000")
+			.send(incVotesObj)
+			.expect(404)
+			.then((res) => {
+				expect(res.body.msg).toBe("Review_id not found");
+			});
+	});
+	test("PATCH - status 400 - Responds with an inc_votes is required error message when passed an empty inc_votes object", () => {
+		const incVotesObj = {};
+		return request(app)
+			.patch("/api/reviews/1")
+			.send(incVotesObj)
+			.expect(400)
```

Comment:
I would recommend changing this test slightly to Status 200. If inc_votes is missing, should not update article and return original single article object